src/Endpoints/AssetFile.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Chromatic\OrangeDam\Endpoints;
+
+class AssetFile extends Endpoint
+{
+    /**
+     * API base path.
+     *
+     * @var string
+     */
+    protected const API_BASE_PATH = '/webapi/mediafile/assetfile/';
+
+    /**
+     * Create links to formatted assets.
+     *
+     * @param string $format
+     *   An Orange Dam asset format, such as TR1, or TR1_WATERMARKED.
+     * @param string $sort
+     *   The sort order of found assets. If you are using an identifer, the
+     *   result should just be one listing. The API default is 'Newest First'.
+     * @param array $queryConditions
+     *   An array of searchkeys and searchvalues with which to perform a Search
+     *   API syntax freetext search.
+     *   Example: ['docType' => 'Image',  'Test.UserFieldA' => 'ZZZ']
+     *
+     * @return \Chromatic\OrangeDam\Http\Response
+     */
+    public function getFormatFile(string $format, string $sort = '', array $queryConditions = [])
+    {
+        // @todo Decide whether we should be validating the sort keys and
```

Comment:
Do we want to decide that now? If it's a low lift, seems like it might be nice to have.

Reply:
I agree it is a nice to have, and the pattern I am seeing elsewhere in this library is to let the request fail at the API endpoint if we pass an invalid parameter. I don't see other endpoints validating anything, so I think I will remove this @todo.